admin/app/components/solidus_admin/products/index/component.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Products::Index::Component < SolidusAdmin::BaseComponent
+  def initialize(products:)
+    @products = products
+  end
+
+  # @!visibility private
```

Comment:
Can you help me understand we need this line, please?

Reply:
There's no strict need here, but it will be used in the next PR when we'll add local helper methods.
This Yard directive will tell that all the methods declared after it will be documented as private.

I'll move the addition to the next PR 👍